### PR TITLE
Clean up parts of the codebase

### PR DESCRIPTION
**Pull request by anonymoose2**
Continuation of #610.

### CODE REVIEW

File: intranet/apps/eighth/views/admin/users.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def list_user_view(request):
 def delete_user_view(request, pk):
     user = get_object_or_404(User, pk=pk)
     if request.method == 'POST':
-        pass
+        return None
```
Comment (theo-o) [inline intranet/apps/eighth/views/admin/users.py]: This should probably raise a 404 not return `None`.

File: intranet/settings/__init__.py [hunk 2]
```diff
@@ -452,7 +452,6 @@ def get_month_seconds():
     "DEFAULT_AUTHENTICATION_CLASSES":
     ("intranet.apps.api.authentication.ApiBasicAuthentication",
      "oauth2_provider.contrib.rest_framework.OAuth2Authentication"),
-    "DEFAULT_PERMISSION_CLASSES": ("rest_framework.permissions.IsAuthenticated",)
```
[the remark this answers is not in the document]
Comment (anonymoose2) [reply intranet/settings/__init__.py]: That key was duplicated. It is also set at the top of the dictionary, and since the values are exactly the same I removed this one.

TL;DR: No functionality change.

[the remark this answers is not in the document]
Comment (theo-o) [reply intranet/settings/__init__.py]: Ah, GitHub didn't show that in the diff.
